### PR TITLE
feature/v5-master-deposit-address

### PR DESCRIPTION
**Pull request by decanus**
https://bybit-exchange.github.io/docs/v5/asset/master-deposit-addr

### CODE REVIEW

**Comment by hirokisan** [comment]
@decanus please import the changes of the main branch into this branch

**Comment by decanus** [comment]
@hirokisan done!

**Comment by hirokisan** [comment]
nits:

It would be helpful if you could also update the documentation.

ref.
https://github.com/hirokisan/bybit/commit/e4d66d7f5d7ff63a6433335d7c33cd78f63b541f

File: v5_asset_service.go [hunk 1]
```diff
@@ -306,6 +307,55 @@ func (s *V5AssetService) GetInternalDepositRecords(param V5GetInternalDepositRec
 	return &res, nil
 }
 
+type V5GetMasterDepositAddressParam struct {
+	Coin  Coin    `json:"coin"`
+	Chain *string `json:"chain,omitempty"`
```
Comment (hirokisan) [inline v5_asset_service.go:312]:
```suggestion
	ChainType *string `json:"chainType,omitempty"`
```

File: v5_asset_service.go [hunk 2]
```diff
@@ -306,6 +307,55 @@ func (s *V5AssetService) GetInternalDepositRecords(param V5GetInternalDepositRec
 	return &res, nil
 }
 
+type V5GetMasterDepositAddressParam struct {
+	Coin  Coin    `json:"coin"`
+	ChainType *string `json:"chainType,omitempty"`
+}
```
Comment (hirokisan) [inline v5_asset_service.go:313]:
```suggestion
type V5GetMasterDepositAddressParam struct {
	Coin  Coin    `url:"coin"`
	ChainType *string `url:"chainType,omitempty"`
}
```

**Review by hirokisan — APPROVED** [review]
Thanks for your contribution!

I will make some adjustments after the merge.

**Pull request merged**